utils/sub8_ros_tools/sub8_ros_tools/math_helpers.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+
+
+def rosmsg_to_numpy(rosmsg, keys=None):
```

Comment:
Might want to be more specific with the function name, since "rosmsg" might suggest you can pass in a msg that is not a ROS vector or a quaternion